{{cookiecutter.project_slug}}/setup.py

```diff
@@ -16,9 +16,39 @@
     {%- if cookiecutter.support_py2 == 'y' %}'six',{%- endif %}
 ]
 
-setup_requirements = ['pytest-runner', ]
+setup_requirements = [
+    'pytest-runner>=2.11.1',
+]
+
+test_requirements = [
+    'coverage>=4.5.1',
+    'pytest>=3.4.2',
+    'tox>=2.9.1',
+]
+
+development_requirements = [
+    # general
+    'bumpversion>=0.5.3',
+    'pip>=9.0.1',
```

Comment:
I suspect that `pip` and `watchdog` are not used and can be safely removed.

Reply:
re offline, watchdog used for `make serve-docs` and pip just provided to ensure minimum dependency